src/domain/event/EventOccurrenceRedirect.tsx

```diff
@@ -0,0 +1,140 @@
+    variables: {
+      id: params.eventId,
+      childId: params.childId,
+    },
+  });
+  const occurrenceResult = useEventOccurrence(
+    params.occurrenceId,
+    params.childId
+  );
+  const location = useLocation();
+  const [copyStatus, setCopyStatus] = useState<CopyState>(CopyStates.initial);
+  const { sendMessage } = useAriaLive();
+
+  const ticketSystem = data?.event?.ticketSystem;
+  const ticketmasterPassword =
+    ticketSystem && 'childPassword' in ticketSystem
+      ? ticketSystem.childPassword
+      : null;
+  const occurrenceTicketSystem =
+    occurrenceResult?.data?.occurrence?.ticketSystem;
+  const ticketmasterUrl =
+    occurrenceTicketSystem && 'url' in occurrenceTicketSystem
+      ? occurrenceTicketSystem.url
+      : null;
+
+  const handlePasswordCopy = () => {
+    if (ticketmasterPassword) {
+      const success = copy(ticketmasterPassword);
+
+      // If copying was successful, true is returned. Otherwise the
+      // copy-to-clipboard package will render a modal which advises the user
+      // to copy by other means.
+      if (success) {
+        sendMessage(t('eventOccurrenceRedirectPage.passwordCopySuccess'));
+        setCopyStatus(CopyStates.success);
+      }
+    }
+  };
+
+  if (loading) {
+    return <LoadingSpinner isLoading={true} />;
+  }
+
+  if (error) {
+    return (
+      <InfoPageLayout
+        title={t('eventOccurrenceRedirectPage.unexpectedError')}
+        description={error.toString()}
+      />
+    );
+  }
+
+  const event = data?.event;
+
+  return (
+    <PageWrapper className={styles.grey}>
+      <div className={styles.wrapper}>
+        <Text as="h1" variant="h2">
+          {t('eventOccurrenceRedirectPage.title')}
+        </Text>
+        <Text variant="body-l">
+          {t('eventOccurrenceRedirectPage.description', {
+            eventName: event?.name,
+          })}
+        </Text>
+        <Text variant="body-l" className={styles.lessMargin}>
+          {t('eventOccurrenceRedirectPage.passwordLabel')}
+        </Text>
+        <div className={styles.row}>
+          <div className={styles.password}>{ticketmasterPassword}</div>
+          <button
+            type="button"
+            onClick={handlePasswordCopy}
+            className={styles.copyButton}
+          >
+            {t('eventOccurrenceRedirectPage.copyPassword')}
+          </button>
+          {copyStatus === CopyStates.success && (
+            <IconCheck className={styles.successCheckMark} />
+          )}
+        </div>
+        <div className={styles.row}>
+          <LinkButton
+            variant="secondary"
+            to={getEventPathname(location.pathname)}
+            className={styles.linkButton}
+          >
+            {t('eventOccurrenceRedirectPage.back')}
+          </LinkButton>
+          <AnchorButton href={ticketmasterUrl}>
+            {t('eventOccurrenceRedirectPage.continue')}
+          </AnchorButton>
+        </div>
```

Comment:
On smaller screen these buttons being on the same line feels little cramped. I haven't seen the designs for this, but in my opinion it would be better if they were under each other and had full width.

Reply:
There was no mobile version of the design. I'll make this change.